plugins/unicodelib/unicodelib.cpp

```diff
@@ -719,6 +720,34 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+void translate(UnicodeString & source, UChar const * sear, unsigned searLen, UChar const * repl, unsigned replLen)
+{
+    UnicodeString search(sear, searLen);
+    UnicodeString replace(repl, replLen);
+    if (search.countChar32() != replace.countChar32() || source.isEmpty() || search.isEmpty() || replace.isEmpty())
+    {
+        return;
+    }
+    StringCharacterIterator it(source);
+    source.remove();
+    int32_t Beg = it.setToStart();
+    while (Beg != it.endIndex())
```

Comment:
I noticed that they were different as well, but forgot to make them consistent after moving onto issues that were actually a problem for the code's execution. The setTo.. functions are meant for code points, and the ..Index() functions are meant for code units; however, they both set the iterator to the same position in the string, and become irrelevant when the move32 function begins to change the position.